### PR TITLE
NSKOPS-179 added settings for build deb package

### PR DESCRIPTION
**Pull request by ghostik**
This PR allows to create deb package elasticsearch-head plugin for ES-2.x.x versions. @sorenmat @tnn

### CODE REVIEW

**Comment by mkostrikin** [comment]
:+1:  looks good
```
docker run -it --rm -v `pwd`:/build  gceranto/nodejs-bower-grunt-yeoman bash
cd /build/
npm install
grunt debian_package --force
```
`Done, but with warnings.`

File: Gruntfile.js [hunk 1]
```diff
@@ -1,115 +1,51 @@
...
-			},
-			lang: {
-				expand: true,
-				cwd: 'src/app/lang/',
-				src: '**',
-				dest: '_site/lang/'
-			}
-		},
-
-		jasmine: {
```
Comment (sorenmat) [inline Gruntfile.js]: why do you remove all this ?

Comment (ghostik) [inline Gruntfile.js]: I created new grunt file with the same name. My target was to build only deb package.

Comment (sorenmat) [inline Gruntfile.js]: But if we need to upgrade this (by pulling in from the upstream) we won't be able to test it. I really think you should just create the deb build as a subtask in the grunt build

Comment (ghostik) [inline Gruntfile.js]: I have remade Gruntfile.js. Please re-review @sorenmat

Comment (mkostrikin) [inline Gruntfile.js]: @ghostik can you take original code and add yours, instead of repear?
bc now its looks even worse them before remade

Comment (ghostik) [inline Gruntfile.js]: Current Gruntfile.js is the original file + my deb task. I only inserted my debian_package task in a begin of all other tasks. Or I don't understand something or you want me to add debian_package task in end all other tasks? @mkostrikin

**Comment by ghostik** [comment]
I have remade the Gruntfile.js with minimal changes form original. @mkostrikin @sorenmat please re-review.

**Comment by mkostrikin** [comment]
Create PR into deb-es2 branch

**Pull request closed without merging**